src/Database/Schema/PropertyDefinition.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace DesignMyNight\Elasticsearch\Database\Schema;
+
+use Illuminate\Support\Fluent;
```

Comment:
I never knew `Fluent` existed - this is great!

Reply:
Yeah one of those magic little gems hidden within the framework, such a cool little idea to populate a key/value array!